src/components/pages/org/OrgPageActions.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { FormattedMessage as Msg } from 'react-intl';
+
+import Link from '../../../common/misc/FormattedLink';
+import LoadingIndicator from '../../../common/misc/LoadingIndicator';
+
+
+export default class OrgPageActions extends React.Component {
```

Comment:
This file should probably not have been included. It's not used anywhere, which makes sense, since we decided to postpone anything related to actions to a later release (and would use `CampaignForm` when we do implement it).